client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -493,12 +518,6 @@ export default function WPCheckout( {
 	);
 }
 
-const CheckoutSummaryArea = styled( CheckoutSummaryAreaUnstyled )`
```

Comment:
Can you explain more about why this is being removed?

Reply:
Well spotted, that is a mistake! I will rollback that change